test/extended/testdata/testjob-plugin.xml

```diff
@@ -53,16 +69,62 @@
       <authToken></authToken>
     </com.openshift.jenkins.plugins.pipeline.OpenShiftServiceVerifier>
 
+    <!-- 1 to 1 mapping -->
     <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
       <apiURL>https://openshift.default.svc.cluster.local</apiURL>
       <namespace>PROJECT_NAME</namespace>
+      <testStream>origin-nodejs-sample</testStream>
       <testTag>latest</testTag>
+      <prodStream>origin-nodejs-sample</prodStream>
       <prodTag>prod</prodTag>
+      <authToken></authToken>
+    </com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+
+    <!-- 1 to N mapping -->
+    <com.openshift.jenkins.plugins.pipeline.OpenShiftImageTagger>
+      <apiURL>https://openshift.default.svc.cluster.local</apiURL>
+      <namespace>PROJECT_NAME</namespace>
       <testStream>origin-nodejs-sample</testStream>
+      <testTag>latest</testTag>
       <prodStream>origin-nodejs-sample</prodStream>
+      <prodTag>prod2,prod3,prod4</prodTag>
```

Comment:
Conversely, I'm also fine with this existing, original test being the one test that vets out the backward compatibility pain.  But when @jupierce pushes the updates for the new test xml files and breaks out the new tests into separate `g.It` sections, they would leverage the new tag.